### PR TITLE
Adding template for nxos show ip interface brief

### PR DESCRIPTION
**Pull request by sidtyagi**
##### ISSUE TYPE
 - New Template Pull Request

### CODE REVIEW

File: tests/cisco_nxos/show_ip_interface_brief/cisco_nxos_show_ip_interface_brief.parsed [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+parsed_sample:
```
Comment (Sandeepsr) [inline tests/cisco_nxos/show_ip_interface_brief/cisco_nxos_show_ip_interface_brief.parsed:2]: Can you please exclude this from parsing ?

File: templates/cisco_nxos_show_ip_interface_brief.template [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Value INTERFACE (\S+)
+Value IP (\S+)
+Value STATUS (\S+)
```
Comment (Sandeepsr) [inline templates/cisco_nxos_show_ip_interface_brief.template:3]: Can we confirm that STATUS will always be a single continues line/word, else you can put (.+)

Comment (sidtyagi) [reply templates/cisco_nxos_show_ip_interface_brief.template:3]: yes, singe line for NxOS (checked 3k and 9k)

File: tests/cisco_nxos/show_ip_interface_brief/cisco_nxos_show_ip_interface_brief.parsed (hunk 1 above)
Comment (sidtyagi) [reply tests/cisco_nxos/show_ip_interface_brief/cisco_nxos_show_ip_interface_brief.parsed]: done

**Pull request merged**